src/xunit.runner.utility/Project/XunitFilters.cs

```diff
@@ -53,9 +60,22 @@ public bool Filter(ITestCase testCase)
                 return false;
             if (!FilterExcludedTraits(testCase))
                 return false;
+            if (!FilterIncludedNameSpaces(testCase))
+                return false;
 
             return true;
         }
+        bool FilterIncludedNameSpaces(ITestCase testCase)
+        {
+            // No assemblies in the filter == everything is okay
+            if (IncludedNameSpaces.Count == 0)
+                return true;
+
+            if (IncludedNameSpaces.Count != 0 && IncludedNameSpaces.Any(a => testCase.TestMethod.TestClass.Class.Name.Contains(a)))
```

Comment:
Done. I think using contains gives a bit more flexibility on what you can filter by but this solves my use case. (Re-fixed the title of the pull request as well, i got a little ahead of myself on the changes)